src/Symfony/Component/ErrorHandler/ErrorRenderer/HtmlErrorRenderer.php

```diff
@@ -92,6 +86,20 @@ public function getStylesheet(): string
         return $this->include('assets/css/exception.css');
     }
 
+    /**
+     * @internal
```

Comment:
An internal API is not covered by semver, making it unsafe to use it in a different component when using semver constraints.